client/src/linkManager.js

```diff
@@ -278,7 +278,7 @@ export class linkManager extends LuigiClientBase {
    */
   fromClosestContext() {
     const hasParentNavigationContext =
-      this.currentContext && this.currentContext.context.parentNavigationContexts.length > 0;
+      this.currentContext && this.currentContext.context?.parentNavigationContexts?.length > 0;
```

Comment:
This line might be even shorter:
`this.currentContext?.context?.parentNavigationContexts?.length`